src/main/java/org/terasology/projectile/ProjectileAuthoritySystem.java

```diff
@@ -127,8 +127,8 @@ public void update(float delta) {
             ProjectileActionComponent projectile = entity.getComponent(ProjectileActionComponent.class);
             ProjectileMotionComponent projectileMotion = entity.getComponent(ProjectileMotionComponent.class);
 
-            if(projectileMotion.distanceTravelled >= projectile.maxDistance && projectile.maxDistance != -1) {
-                if(projectile.reusable)
+            if (projectileMotion.distanceTravelled >= projectile.maxDistance && projectile.maxDistance != -1) {
+                if (projectile.reusable)
                     entity.send(new DeactivateProjectileEvent());
                 else
                     entity.destroy();
```

Comment:
Could you add brackets even for 1-line code? that would be best for visibility.

On the distance side of things, I don't think I quite understand what `projectile.maxDistance - projectileMotion.distanceTravelled` would represent. It's the remaining distance to the edge of where it would be unspawned, the displacement would automatically be added for the next tick and thus the arrow removed then. If that's what's intended, then it's all good